content/en/blog/_posts/2023-10-18-gateway-api-time-to-switch-i2gw/index.md

```diff
@@ -0,0 +1,203 @@
+---
+layout: blog
+title: "Time to Upgrade; Migrating to Gateway API"
+date: 2023-10-24T10:00:00-08:00
+slug: introducing-ingress2gateway
+---
+
+***Authors:*** Lior Lieberman (Google), Kobi Levi (independent)
+
+Today we are releasing [ingress2gateway](https://github.com/kubernetes-sigs/ingress2gateway), a tool
+that can help you migrate from [Ingress](/docs/concepts/services-networking/ingress/) to [Gateway
+API](https://gateway-api.sigs.k8s.io). Gateway API is just weeks away from graduating to GA, if you
```

Comment:
weeks? Or days?

Reply:
resolving this and leave the first comment open